1-js/13-modules/01-modules-intro/article.md

```diff
@@ -229,162 +229,162 @@ Compare it to non-module scripts, where `this` is a global object:
 
 ```html
-<!-- Assuming we got bundle.js from a tool like Webpack -->
+<!-- با فرض اینکه ما bundle.js را از یک ابزار مانند Webpack گرفته ایم -->
 <script src="bundle.js"></script>
 ```
 
-That said, native modules are also usable. So we won't be using Webpack here: you can configure it later.
+با این حساب، ماژول ها بومی و نیتیو هم قابل استفاده هستند. در نتیجه ما از Webpack در اینجا استفاده نمی کنیم: شما می توانید آن را در آینده تنظیم کنید.
 
-## Summary
+## خلاصه
 
-To summarize, the core concepts are:
+برای خلاصه کردن، مفاهیم اصلی اینها هستند:
 
-1. A module is a file. To make `import/export` work, browsers need `<script type="module">`. Modules have several differences:
-    - Deferred by default.
-    - Async works on inline scripts.
-    - To load external scripts from another origin (domain/protocol/port), CORS headers are needed.
-    - Duplicate external scripts are ignored.
-2. Modules have their own, local top-level scope and interchange functionality via `import/export`.
-3. Modules always `use strict`.
-4. Module code is executed only once. Exports are created once and shared between importers.
+1. هر ماژول یک فایل است. برای اینکه عبارات `import/export` کار بکنند، مرورگرها نیاز به `<script type="module">` دارند. ماژول های چندین تفاوت با اسکریپت های معمولی دارند:
+    - به صورت پیش فرض به تعویق افتاده (Deferred) هستند.
+    - در اسکریپت های inline Async جواب می دهد.
+    - برای بارگزاری اسکریپت های خارجی(external) از منابع دیگر (دامنه/پروتکل/پورت)، هدر های CORS نیاز هستند.
+    - اسکریپت های مشابه external نادیده گرفته می شوند.
+2. ماژول های اسکوپ سطح بالای خود را دارند و از طریق عبارات ‍`import/export` کارایی های خود را با دیگر اسکریپت های به اشتراک می گذارند.
+3. ماژول ها همیشه در حالت ‍`use strict` هستند.
+4. کد ماژول ها تنها یک مرتبه اجرا می شوند. Exportها تنها یک مرتبه ساخته شده و سپس بین تمام importer ها به اشتراک گذاشته می شوند.
 
-When we use modules, each module implements the functionality and exports it. Then we use `import` to directly import it where it's needed. The browser loads and evaluates the scripts automatically.
+وقتی که ما از یک ماژول استفاده می کنیم، هر ماژول یک کارایی را بوجود آورده و ان را صادر می کند. سپس ما از عبارت `import` برای مستقیما وارد کردن ماژول به جایی که به آن نیاز داریم، استفاده می کنیم. مرورگر به صورت خودکار اسکریپت را بارگذاری و ارزیابی می کند.
 
-In production, people often use bundlers such as [Webpack](https://webpack.js.org) to bundle modules together for performance and other reasons.
+در زمان انتشار، برنامه نویسان معمولا از باندل هایی مانند [Webpack](https://webpack.js.org) برای جمع کردن ماژول ها در کنار هم و بالا بردن کارایی و چند دلیل دیگر استفاده می کنند.
 
-In the next chapter we'll see more examples of modules, and how things can be exported/imported.
+در فصل بعد ما مثال های بیشتری از ماژول ها را می بینیم، و اینکه چگونه آن ها صادر/وارد می شوند.
```

Comment:
As you didn't translate import/export in other places, please don't translate it here as well.